internal/plugin/grpc_client_plugin.go

```diff
@@ -297,9 +306,60 @@ func (gc *GrpcClient) createConnectionRequest(resource *v1.Resource) (*v1.Create
 func (gc *GrpcClient) Subscriptions() []string {
 	return []string{
 		bus.ResourceTopic,
+		bus.InstanceHealthTopic,
 	}
 }
 
+// says sendDataPlaneHealthUpdate & sendDataPlaneStatusUpdate are duplicate code
+// nolint: dupl
+func (gc *GrpcClient) sendDataPlaneHealthUpdate(ctx context.Context, instanceHealths []*v1.InstanceHealth) error {
+	if !gc.isConnected.Load() {
+		slog.InfoContext(ctx, "gRPC client not connected yet. Skipping sending data plane health update")
+		return nil
+	}
```

Comment:
This is likely out of scope but should we be buffering messages in the event that we aren't connected?

Reply:
I'm not sure, to be honest I just went off what was done for the data plane status for this.